index.html

```diff
@@ -3882,6 +3912,21 @@ <h2>JsonWebKey Examples</h2>
   "capabilityInvocation": [
     "https://controller.example/123#key-2"
   ]
+}
+          </pre>
+
+          <pre class="example nohighlight"
+            title="An SM2 public key encoded as a JsonWebKey">
+{
+  "id": "https://jsonwebkey.example/issuer/123#key-0",
+  "type": "JsonWebKey",
+  "controller": "https://jsonwebkey.example/issuer/123",
+  "publicKeyJwk": {
+    "kty": "EC",
+    "crv": "SM2",
+    "x": "NIMpjy0M25izi2DrQLhQBlL8b45Lt9a0FzfELdpdO0s",
+    "y": "NXstrSPtSp1cZzvupZ8C68FCgSQhZbuGYLorP6F1N2w"
+  }
```

Comment:
```suggestion
```

@selfissued noted that JsonWebKey does not officially support SM2 and has requested its removal so this PR can be merged.